app/models/game.rb

```diff
@@ -52,4 +52,23 @@ def in_check?(color)
     end
     false
   end
+
+  def checkmate?(color)
+    king = Piece.where(color: color, type: 'King').first
```

Comment:
This line isn't being used anymore, so you can just remove it.

Reply:
Yes I can take care of that.